correct_periscope_drift.py

```diff
@@ -0,0 +1,431 @@
+
+    # open_param_file, from ciao_contrib.param_wrapper,
+    # opens the parameter file and sorts out the command
+    # line. It returns a dictionary with a few fields;
+    # fp is the handle to use in paramio calls, as shown
+    # below; and parnams is the name to use if you need
+    # to re-open the parameter file to set things
+    # (which is unlikely, speak to Doug if you think you
+    # might need this).
+    #
+    pinfo = open_param_file(argv, toolname=TOOLNAME)
+    fp = pinfo["fp"]
+
+    # Use the parameter library to get the arguments
+    # and perform any desired validity checks (e.g.
+    # the outfile check/clobber check shown below).
+    #
+    # Parameters should be queried in the same order
+    # as the parameter file
+    #
+    mypars = {'progname': pinfo['progname'],
+              'parname': pinfo['parname']}
+
+    for stringpar in ['infile', 'evtfile',
+                      'outfile', 'corr_plot_prefix']:
+        mypars[stringpar] = pio.pgetstr(fp, stringpar)
+        if mypars[stringpar].strip() == "":
+            raise ValueError("{} parameter is empty".format(stringpar))
+
+    for floatpar in ['x', 'y', 'radius', 'src_min_counts']:
+        mypars[floatpar] = pio.pgetd(fp, floatpar)
+
+    mypars['corr_poly_degree'] = pio.pgeti(fp, "corr_poly_degree")
+
+    clobber = pio.pgetb(fp, "clobber")
+    verbose = pio.pgeti(fp, "verbose")
+
+    # We close the parameter file here; if you need to write
+    # values to the file you can either leave it open or -
+    # possibly better - is to close it and re-open it later.
+    #
+    pio.paramclose(fp)
+
+    # Set tool and module verbosity
+    set_verbosity(verbose)
+
+    # check outfile, exiting if it exists and clobber is
+    # not set. Note: the error message should be updated to better
+    # match that used by CIAO tools but I have not done this.
+    #
+    #
+    if not clobber and os.path.exists(mypars['outfile']):
+        raise IOError("clobber is no and outfile ({0}) exists".format(
+                mypars['outfile']))
+
+    mypars['clobber'] = clobber
+    mypars['verbose'] = verbose
+
+    # Return a dictionary with useful info (you could return a tuple or
+    # something else, depending on the parameter file)
+    #
+    return mypars
+
+
+# Display parameter info to the user.
+# The format and choice of information to display is up to you;
+# I've chosen to include some ancilalry information but it
+# depends on the tool what should be used.
+#
+def display_start_info(opts):
+    v1("Running: {0}".format(opts["progname"]))
+    v2("  version = {0}".format(VERSION))
+    v2("with parameters:")
+    v2("  infile={0}".format(opts["infile"]))
+    v2("  evtfile={0}".format(opts["evtfile"]))
+    v2("  outfile={0}".format(opts["outfile"]))
+    # probably other values here too
+    v2("  verbose={0}".format(opts["verbose"]))
+    v2("  and CALDB is set to  {0}".format(os.environ["CALDB"]))
```

Comment:
Suggest dropping `CALDB` from the verbose output since you're not explicitly using any CALDB files here.